### ISSUE TITLE
Feature request: Subtracting costs

### ISSUE THREAD

**Issue by syphernl** [infracost/infracost#2360]
For one of our projects we are setting up a zonal GKE cluster, of which one is free under the GCP tier (as is 1 autopilot cluster).
However, as Infracost is not aware of the amount of (zonal) GKE clusters it wouldn't know about that and still indicates that the Cluster Management Fee is $73 per month rather than $0. This significantly increases the price estimation.

Is there a way to subtract a specific amount from the estimation Infracost does? 

For instance:
* The cluster costs $73
* The instance cost $7 per month

In this case Infracost estimates the total costs to be $80.
I'd like to subtract $73 making the total costs only $7.

**Comment by syphernl** [comment]
> Perhaps we could have a value in the [Infracost usage](https://www.infracost.io/docs/features/usage_based_resources/#3-run-with-usage-file) for this resource, like `is_free_tier: true` so you could specify it. Would that solution work for you?

Yes, that would work just fine! It is even better because it would prevent manual changes of the subtraction amount later on in case the cloud pricing changes.

**Comment by glenngillen** [comment]
We're moving to a contribution model where Issues will be exclusively for work that is ready to be worked on. Discussions and feature requests about potential future capabilities will be over in Discussions. There has been very little community or customer interest in this capability for an extended period, but if it still interests you (or you're now reading this issue because you've been looking for a solution) please create a new thread in Discussions where we can better gather interest from the community.
